xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/pom.xml

```diff
@@ -94,6 +94,23 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+            <!-- We keep the Vue files and the build configuration files in separate folders in the source tree but
+              we need them in the same folder when building the Vue components. -->
```

Comment:
This was copied from above, but I think it needs to be updated.

Reply:
ok, it don't think we need to merge this PR, but I'll reuse this codebase when providing tests for the next evolutions. I'll also wait for @ClemDee input since he spent more time than us on it :)

Reply:
I think we should merge it. But sure, we can wait for @ClemDee opinion first.